Options.py

```diff
@@ -1076,14 +1076,14 @@ def verify(self, world: typing.Type[World], player_name: str, plando_options: "P
 
 @dataclass
 class PerGameCommonOptions(CommonOptions):
-    local_items: LocalItems
-    non_local_items: NonLocalItems
-    start_inventory: StartInventory
-    start_hints: StartHints
-    start_location_hints: StartLocationHints
-    exclude_locations: ExcludeLocations
-    priority_locations: PriorityLocations
-    item_links: ItemLinks
+    local_items: LocalItems = LocalItems.default
```

Comment:
This doesn't make sense.
`LocalItems.default` is not an instance of `LocalItems`

I see you trying to use the default options in unit tests.
I think @alwaysintreble might be able to show a better way to get an instance of the dataclass with all the defaults for unit tests.

What I did to get an instance of the options dataclass for testing is: set up a world with `WorldTestBase` and then just pull the options from that.
But, admittedly, that is more work than should be necessary just to get an instance with the default options.

Reply:
I don't have any idea what the goal is here, but to create an instance of an options dataclass you'd just mimic the behavior we already do in `multiworld.set_options()`